services/graphite/service_test.go

```diff
@@ -221,6 +230,152 @@ func Test_DecodeMetric(t *testing.T) {
 	}
 }
 
+func Test_ServerGraphiteTCP(t *testing.T) {
+	t.Parallel()
+	if testing.Short() {
+		t.Skip()
+	}
```

Comment:
Is this test long? I don't think we need the skip here or in the other tests. Especially since they're marked parallel now.